registry/handlers/referrers.go

```diff
@@ -190,3 +177,57 @@ func readlink(ctx context.Context, path string, stDriver driver.StorageDriver) (
 
 	return digest.Parse(string(content))
 }
+
+func getArtifactReferrers(ctx context.Context,
+	blobStatter distribution.BlobStatter,
+	referrerDigest digest.Digest,
+	man distribution.Manifest,
+	referrers *[]v1.Descriptor,
+	artifactType string) error {
```

Comment:
Instead of the code pattern
```go
func doSomething(result *[]any)
```
Try golang-style
```go
func doSomething(result []any) []any
```

Reply:
In fact, you don't even need to pass the `referrers`. All you want to return is a referrer descriptor.

Think about
```go
func generateReferrerFromArtifact(ctx context.Context, blobStatter distribution.BlobStatter, referrerDigest digest.Digest, man *ociartifact.DeserializedManifest, artifactType string) (v1.Descriptor, error)
```

You could apply the same thing to `getImageReferrers`.

Reply:
Good point. Changed accordingly.